### PR TITLE
Correct grouping name Conover

### PR DESCRIPTION
**Pull request by JohnnyDoorn**
Fix https://github.com/jasp-stats/jasp-test-release/issues/2304

### CODE REVIEW

File: R/anovarepeatedmeasures.R [hunk 1]
```diff
@@ -1587,7 +1587,7 @@ AnovaRepeatedMeasuresInternal <- function(jaspResults, dataset = NULL, options)
 
     conoverTable <- createConoverTable(groupingVar)
     noteBlockName <- ifelse(blockingVar == .BANOVAsubjectName, "subject", .BANOVAsubjectName)
```
Comment (maltelueken) [inline R/anovarepeatedmeasures.R:1589]: I still got the same bug when no grouping variable was selected. I think this fix should do the trick:

```suggestion
    noteBlockName <- ifelse(blockingVar == .BANOVAsubjectName, "subject", blockingVar)
    conoverTable$addFootnote(gettextf("Grouped by %s.", noteBlockName))
```

**Comment by maltelueken** [comment]
Some unit tests seem to fail because of issues with installing jaspBase. Is this expected?

**Comment by Kucharssim** [comment]
> with installing jaspBase. Is this expected?

It is not.

@JohnnyDoorn can you make the changes based on the review by @maltelueken, hopefully the tests will pass now.

**Comment by maltelueken** [comment]
All the tests seem to work except one. @Kucharssim is that still due to some unrelated issue?

**Review by maltelueken — APPROVED** [review]

**Review by Kucharssim — APPROVED** [review]
> All the tests seem to work except one. @Kucharssim is that still due to some unrelated issue?

Yes, @JohnnyDoorn that seems to be the bootstrapping test on ubuntu we talked about some time ago. I'll merge this to leave the fix of this test for a separate PR.

**Pull request merged**